protocol/src/main/contraband-scala/sbt/protocol/codec/ExectionEventFormats.scala

```diff
@@ -5,21 +5,21 @@
 // DO NOT EDIT MANUALLY
 package sbt.protocol.codec
 import _root_.sjsonnew.{ deserializationError, serializationError, Builder, JsonFormat, Unbuilder }
-trait ExectionEventFormats { self: sjsonnew.BasicJsonProtocol =>
-implicit lazy val ExectionEventFormat: JsonFormat[sbt.protocol.ExectionEvent] = new JsonFormat[sbt.protocol.ExectionEvent] {
-  override def read[J](jsOpt: Option[J], unbuilder: Unbuilder[J]): sbt.protocol.ExectionEvent = {
+trait ExecutionEventFormats { self: sjsonnew.BasicJsonProtocol =>
```

Comment:
This is a generated code, and an outdated one. This file should be removed since there's now https://github.com/sbt/sbt/blob/1.0.x/protocol/src/main/contraband-scala/sbt/protocol/codec/ExecutionEventFormats.scala